src/lib/integration_test_cloud_engine/coda_automation.ml

```diff
@@ -311,31 +313,43 @@ module Network_manager = struct
   let run_cmd_exn t prog args = run_cmd_exn t.testnet_dir prog args
 
   let create ~logger (network_config : Network_config.t) =
+    let%bind all_namespaces_str =
+      Cmd_util.run_cmd_exn "/" "kubectl"
+        ["get"; "namespaces"; "-ojsonpath={.items[*].metadata.name}"]
+    in
+    let all_namespaces = String.split ~on:' ' all_namespaces_str in
+    let rec prompt_continue () =
```

Comment:
This is very similar to the `prompt_continue` from `test_executive.ml`. Maybe we should move this into a util somewhere in `integration_test_lib`? Eg, `Integration_test_lib.Cli_util.prompt_continue`?